content/en/security/application_security/threats/threat-overview.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Threat Overview
+kind: documentation
+aliases:  
+---
+
+{{< img src="security/application_security/threats/appsec-threat-overview-page-top.png" alt="Screenshot of the ASM Threat Overview page"  >}}
+
+## Introduction
+
+The ASM Threats Overview provides an at a glance view of your application and API posture, highlighting trends, service exposure, attack traffic, and the impact on business logic. Users can quickly pivot from widgets to related traces.
+
+The Threats Overview is divided into several sections, each focusing on a different aspect of security with supporting information.
+
+## Sections
```

Comment:
Consider formatting these sections as a description list. I think it would look nice and improve readability. See https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Description-lists for the Markdown and an example.